### PR TITLE
Fix: comment handling for database in dbc format

### PR DESCRIPTION
**Pull request by malneni**
**Updated dbc.py** to handle comment for Bus.
For comment Sequence after CM_ is choice. ex: CM_ "Special Release";

Test: 
**Added comment to vehicle.dbc.**,  Existing tests to compare InDBC = OutDBC is enough for testing.

### CODE REVIEW

File: cantools/database/can/formats/dbc.py [hunk 1]
```diff
@@ -381,6 +383,18 @@ def attributes(self):
...
+        """The DBC specific comment as a string.
+
+        """
+
+        return self._comment
+
+    @comment.setter
+    def comment(self, value):
+        self._comment = value
+
```
Comment (eerimoq) [inline cantools/database/can/formats/dbc.py:397]: What's the reason for adding the comment attribute to the DbcSpecifics class? Isn't it better to use the comment attribute on the Bus class?

File: cantools/database/can/formats/dbc.py [hunk 2]
```diff
@@ -558,6 +572,13 @@ def _dump_senders(database):
 def _dump_comments(database):
     cm = []
 
+    if database.dbc is not None:
+        if database.dbc.comment is not None:
+            cm.append(
+                'CM_ "{comment}";'.format(
+                    #comment=database.dbc.comment.replace('"', '\\"')))
+                    comment=database.dbc.comment) )
```
Comment (eerimoq) [inline cantools/database/can/formats/dbc.py:580]: Remove commented code.

Comment (malneni) [reply cantools/database/can/formats/dbc.py:580]: 1. Tests: OK, I will add more tests.
2. I do not have specific reason between DbcSpecifics class vs Bus class. It is tricky to choose
I tried to duplicate comments and see how DBC viewer tool handles like
CM_ "Special Release";
CM_ "Special Release2";
tool combines them like: CM_ "Special ReleaseSpecial Release2";
& Bus class handles only 3 attributes, if  there is no Bus name and baudrate attribute in dbc file. comment  is only attribute left 

DbcSpecifics class handles attributes dictionary so looks more appropriate for me personally

3.  I will remove commented code

Comment (eerimoq) [reply cantools/database/can/formats/dbc.py:580]: Use the comment attribute in the Bus class. Do not add anything to the DbcSpecifiers class. The reason is that a comment seems generic over multiple database formats, and therefore belongs to the Bus class.

Comment (eerimoq) [reply cantools/database/can/formats/dbc.py:580]: It looks like only the last found CM_ for signals and messages are stored. I suggest using the same logic here. That is, not concatenating comments.

**Comment by malneni** [comment]
I am redoing the update on latest release

**Pull request closed without merging**